src/Pages/Signup/index.js

```diff
@@ -1,7 +1,258 @@
+                      지켜주세요.
+                    </li>
+                    <li className="emailCautionList">
+                      ・ 생성한 카카오메일로 카카오계정과 관련한 알림을 받아볼
+                      수 있습니다.
+                    </li>
+                  </ul>
+                </div>
+                <div className="pw">
+                  <strong className="pwTitle">비밀번호</strong>
+                  <div className="pwBox">
+                    <label htmlFor="pwInput" className="pwLabel">
+                      <input
+                        autoComplete="off"
+                        className="pwInput"
+                        type="password"
+                        name="password"
+                        placeholder="비밀번호(8~32자리)"
+                        onChange={this.handleInput}
+                      />
+                    </label>
+                  </div>
+                </div>
+                <div className="nickname">
+                  <strong className="nicknameTitle">닉네임</strong>
+                  <div className="nicknameBox">
+                    <label htmlFor="nicknameInput" className="nicknameLabel">
+                      <input
+                        autoComplete="off"
+                        className="nicknameInput"
+                        type="text"
+                        name="nickname"
+                        placeholder="닉네임을 입력해 주세요."
+                        onChange={this.handleInput}
+                      />
+                    </label>
+                  </div>
+                </div>
+                <div className="phone">
+                  <strong className="phoneTitle">전화번호</strong>
+                  <div className="phoneBox">
+                    <label htmlFor="phoneInput" className="phoneLabel">
+                      <input
+                        autoComplete="off"
+                        className="phoneInput"
+                        type="text"
+                        name="phone_number"
+                        placeholder="전화번호를 입력해 주세요."
+                        onChange={this.handleInput}
+                      />
+                    </label>
+                  </div>
+                </div>
+                <div className="birthday">
+                  <strong className="birthdayTitle">생일</strong>
+                  <div className="birthdayBox">
+                    <label className="birthdayLabel">
+                      <div className="dateBox">
+                        <div className="yearSelect">
+                          <select
+                            name="year"
+                            id="birthYear"
+                            onChange={this.handleInput}
+                          >
+                            <option value="">연도</option>
+                            {BIRTH_YEARS.map((el) => {
+                              return (
+                                <option value={el} key={el}>
+                                  {el}
```

Comment:
```suggestion
                            {BIRTH_YEARS.map((year) => {
                              return (
                                <option value={year} key={year}>
                                  {year}
```
el, data 같은 이름보다 year처럼 구체적인 이름이 좋아보입니다